frontend/.eslintrc.js

```diff
@@ -4,16 +4,11 @@ module.exports = {
     node: true,
   },
   extends: [
-    "plugin:vue/essential",
-    "eslint:recommended",
-    "@vue/typescript/recommended",
-    "plugin:prettier/recommended",
+    'plugin:vue/vue3-essential',
```

Comment:
better to use recommended https://eslint.vuejs.org/user-guide/#bundle-configurations

Reply:
done in other branch

Reply:
done in other branch